docs/workflow/building/coreclr/linux-instructions.md

```diff
@@ -157,6 +153,6 @@ Running a single test
 After `src/coreclr/build-test.sh` is run, corerun from the Core_Root folder is ready to be run. This can be done by using the full absolute path to corerun, or by setting an environment variable to the Core_Root folder.
 
 ```sh
-export CORE_ROOT=~/runtime/artifacts/tests/coreclr/Linux.x64.Debug/Tests/Core_Root
+export CORE_ROOT=/runtime/artifacts/tests/coreclr/Linux.x64.Debug/Tests/Core_Root
```

Comment:
this is weird. I don't think any Unix user would place the runtime repo on the machine root.

Reply:
Normally, no, but all our container builds and the docker instructions mount the repo there.

Reply:
@ViktorHofer are you ok with this? I don't feel strongly. I was just aligning to the instructions for the Docker mount.

Reply:
sure.